opsdroid/opsdroid
# WIP hangouts connector

Early stages of a Google Hangouts Connector

- [x] Connect to hangups client
- [ ] Receive msgs
- [ ] Respond to msgs

To test:

Add to configuration.yaml:
```yaml
connectors:
  - name: hangups
    no-cache: true
    path: [...repository path...]/opsdroid/opsdroid/connector/hangouts
```

After starting opsdroid comes up it will ask to authenticate with a google account and then you should see all of the chat events start streaming in opsdroid debug logs.
